src/theme/styledTheme.js

```diff
@@ -28,6 +28,7 @@ const theme = {
   },
 
   fontSize: {
+    xxxs:'0.875rem',
```

Comment:
I'm a bit confused as to why you're using such a small font-size. I think the lowest we should go is 1rem.

Reply:
if we use a bigger font it looks weird but we change it later on if it doesn't look proper